seedDatabase/query_PostgreSQL.sql

```diff
@@ -0,0 +1,73 @@
+DROP TABLE courses;
+DROP TABLE sections;
+DROP TABLE [elements];
+
+CREATE TABLE courses (
+	course_id SERIAL,
+	id VARCHAR PRIMARY KEY,
+	course_length TIMESTAMP,
+	total_sections INTEGER,
+	total_lectures INTEGER,
+	total_exercises INTEGER,
+	total_articles INTEGER,
+	total_quizzes INTEGER,
+	updated_at DATE
+);
+
+CREATE INDEX idx_course ON courses (course_id, id);
```

Comment:
way to index things. My queries were pretty quick, but if they were slower, I think this would have been my first option.